app/controllers/provider_interface/applications_exports_controller.rb

```diff
@@ -5,7 +5,10 @@ class ApplicationsExportsController < ProviderInterfaceController
     def export
       respond_to do |format|
         format.csv do
-          csv_data = HesaDataExport.new(provider_ids: current_provider_user.providers.map(&:id)).call
+          csv_data = HesaDataExport.new(
+            provider_ids: current_provider_user.providers.map(&:id),
+            actor: current_provider_user,
```

Comment:
We can just pass in actor here to simplify the interface.

When I did that and edited the tests accordingly, I realised that currently only members of the training provider can generate an export with applications to their courses (since we check the application choice `provider` and not the `accredited_provider`).

Do we want to expand to allow ratifiers to generate this export?